### DISCUSSION TITLE
OBS Virtual Camera is not working on WEBEX meetings

### DISCUSSION THREAD

**Discussion by jemedgreat** [obsproject/obs-studio#7894] [General]
I frequently using my OBS virtual camera in Webex but just suddenly it is not working anymore. The Webex app says that the virtual cam isn't connected.

**Comment by RytoEX** [comment]
This sounds more likely to be a feature request, and we are not currently accepting feature requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [ideas page](https://ideas.obsproject.com) for feature submissions.

Thank you!
